packages/lesswrong/components/common/HeaderSubtitle.tsx

```diff
@@ -2,12 +2,13 @@ import React from 'react';
 import { Components, registerComponent } from '../../lib/vulcan-lib';
 import { useSubscribedLocation } from '../../lib/routeUtil';
 import { Link } from '../../lib/reactRouterWrapper';
+import { isEAForum } from '../../lib/instanceSettings';
 
 export const styles = (theme: ThemeType): JssStyles => ({
   subtitle: {
     marginLeft: '1em',
     paddingLeft: '1em',
-    textTransform: 'uppercase',
+    textTransform: isEAForum ? undefined : 'uppercase',
```

Comment:
@agnesstenlund Just wanted to flag this other small change I made. With the shorter site header, the uppercase subsection text looked quite loud to me:
<img width="1439" alt="Screen Shot 2023-06-30 at 2 01 09 PM" src="https://github.com/ForumMagnum/ForumMagnum/assets/9057804/ca3c9bfd-a540-4844-8c5c-e84587381bd0">

-----
I felt it looked better with sentence casing:
<img width="1439" alt="Screen Shot 2023-06-30 at 2 01 19 PM" src="https://github.com/ForumMagnum/ForumMagnum/assets/9057804/74c9457e-1133-4b57-ab6e-8b0f7e996e2d">